src/Honeybee_ExportEPC.py

```diff
@@ -1,20 +1,29 @@
-# By "Write your names here..."
-# "Write your email address here
+# By Patrick Chopson and Sandeep Ahuja
+# patrick.chopson@perkinswill.com and sandeep.ahuja@perkinswill.com
 # Honeybee started by Mostapha Sadeghipour Roudsari is licensed
 # under a Creative Commons Attribution-ShareAlike 3.0 Unported License.
 
+##!!!!!!Download the latest EPC (Monthly) and EPC Weather Converter from - http://zeroenergygreen.com/downloads/
```

Comment:
I would add this to description of the component so users can see it without opening the component. Also you may want to add a check and make sure the user have it already on their system and it is open, and otherwise give them a warning and return.